app/interactors/calificationInteractor.js

```diff
@@ -0,0 +1,25 @@
+const calificationService = require('../services/calificationService');
+const CalificationModel = require('../models').calification;
+const weetService = require('../services/weetsService');
+const userService = require('../services/userService');
+const errorMessages = require('../constants/errorMessages');
+const { notFoundError } = require('../errors');
+
+exports.rateWeet = async body => {
+  const ratingUser = await userService.findUserByEmail(body.email);
+  if (!ratingUser) throw notFoundError(errorMessages.userNotFound);
+
+  const weet = await weetService.findWeetById(body.weetId);
+  if (!weet) throw notFoundError(errorMessages.weetNotFound);
+
+  const weetAuthor = await userService.findUserById(weet.userId);
+  if (!weetAuthor) throw notFoundError(errorMessages.weetAuthorNotFound);
```

Comment:
Those 3 database requests could be parallel, what do you think?

Reply:
Yes, great Idea! 
I can run two of those because the last one need the weet result (for the weet.userId)